simulation_parameters/Constants.cs

```diff
@@ -575,6 +577,9 @@ public static class Constants
             0 :
             -42;
 
+    private const uint MinimumRunnableProcessFractionIsAboveEpsilon =
```

Comment:
I'm all for hitchhikers guide references, but is this ever used?

Reply:
This will actually prevent the code from compiling if the constant is wrong.
C# doesn't have static assert (like C++ does), so this is the only way to implement static asserts. See the one above this one, there's already one of these kinds of "static asserts" in the code.